selfservice/flow/settings/error.go

```diff
@@ -163,9 +166,13 @@ func (s *ErrorHandler) WriteFlowError(
 		if shouldRespondWithJSON {
 			s.d.Writer().WriteError(w, r, aalErr)
 		} else {
-			http.Redirect(w, r, urlx.CopyWithQuery(
-				urlx.AppendPaths(s.d.Config().SelfPublicURL(r.Context()), login.RouteInitBrowserFlow),
-				url.Values{"aal": {string(identity.AuthenticatorAssuranceLevel2)}}).String(), http.StatusSeeOther)
+			if aalErr.RedirectTo == "" {
```

Comment:
I could not find a place (except for in some tests) where this value is empty. Do we need this code change?

Reply:
nice catch! I think this was code added at some point while testing this and forgot to remove it.